includes/User.php

```diff
@@ -1373,6 +1373,19 @@ private function getBlockedStatus( $bFromSlave = true, $shouldLogBlockInStats =
 	}
 
 	/**
+	 * SUS-1649: Clear all block-related info from this instance of User class
+	 * This allows us to re-run block check functions with different parameters (e.g. checking for only local blocks)
+	 * @see User::getBlockedStatus()
+	 */
+	public function clearBlockInfo() {
```

Comment:
Let's add an information here that this is a custom method added by us - a simple `Wikia change` will do :)